docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
+**Staking** tokens is also referred to as **delegating**.
 
-A validator can **self-delegate only through the CLI** – explorer v1 doesn’t provide this functionality.
+:::
 
-A validator can also have an **additional stake added** to it via the CLI to increase its voter power.
+A **Validator Operator** must **self-delegate** at least 2M CUDOS to a node for it to be operational. They can increase their chances of attracting delegations from token holders by running a highly reliable node with excellent uptime. This in turn increases their chances of being selected to issue a block. 
 
-## Block Rewards 
+Validators are ranked according to the amount of tokens that have been delegated to them, including self-delegations. 
 
-The more tokens staked to your Validator node, the higher the likelihood of being chosen to propose and sign a block. 
+At each block, the top 100 validators (who are not jailed) are said to be **bonded** and have the following rights: 
 
-* **Block rewards** are calculated after each block, and is equal to the sum of the **staking rewards** and the **tx fees** for that block.
+- To propose or sign blocks.
+- To earn block rewards.
+- To be subject to slashing.
 
-* Staking rewards occur in a single pool. 
+### Jailing/Slashing
 
-* They are distributed as follows:  
-
-    1. Validators are assigned rewards proportional to their
-    
-    2. Users who have delegated tokens are required to **actively** claim those rewards by sending appropriate txs to the blockchain.
+Tokens delegated to a bonded validator can be **slashed** due to validator downtime, double signing event or other bad behaviour. 
 
-    3. The **block proposer** gets a base bonus as 1% of the block rewards
-    * In addition, it gets (4*P)% percent of the block rewards where P=(total power of validators with included precommits / total bonded validator power)
-    * This means it gets an additional bonus between 2.67% and 4%
-    * This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.
+#### Downtime event
 
-    4. 20% goes to the community pool
-    5. The rest is distributed to all of the bonded validators (including the block proposer) proportional to their voting power, regardless of them signing the block.
+If a validator fails to sign 90%  of blocks within a 19200-block interval (i.e. 17280 blocks), then it is jailed and the following occurs:
 
+    1. It becomes not-bonded and begins unbonding
+    2. It stops signing and proposing blocks
+    3. It stops earning block rewards
+    4. Its stake gets slashed by 0.01%
 
-(In addition, it gets (4*P)% percent of the block rewards where P=(total power of validators with included precommits / total bonded validator power)
-This means it gets an additional bonus between 2.67% and 4%
-This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.)
+The validator must unjail itself as quickly as possible after being jailed. 
```

Comment:
There is a 10-minute waiting period before a validator can unjail itself.

Reply:
done